.github/workflows/maven.yml

```diff
@@ -51,3 +51,20 @@ jobs:
 
       - name: 'Build'
         run: ./mvnw -ntp -B --file pom.xml verify
+        env:
+          DEBEZIUM_TAG: ${{ matrix.debezium_tag }}
```

Comment:
Instead of passing in the Debezium tag, could we make this to use the Kafka version (as that's what matters in the end for controlling which tests to run). E.g. when a test fails due to an unsupported API, we'd easily say that this is about, say, Kafka Connect 3.2, rather than some largely unrelated Debezium version. This would require to codify the Kafka-to-Debezium version mapping somewhere in the code which spins up Testcontainers.

Reply:
Good call, done! I've left in the latest stable and nightly images since they may help us catch breaking changes with Connect sooner, but can remove those too if they don't seem worth the effort.